### PR TITLE
feature: add git-cart script

### PR DESCRIPTION
**Pull request by Nate1729**
# Description

Add a script that goes to the `$ORG_GIT_ROOT` directory and lists all repos in the directory and what current branch you are on.

### CODE REVIEW

File: git-cart.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
```
Comment (garrypolley) [inline git-cart.sh]: What's the usage for this?  

Ideally moves to a `/git-commands` subfolder.

File: git-cart.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+cd $ORG_GIT_ROOT
+printf "| %-20.20s | %-30.30s |\n" "Repo" "Current Branch"
```
Comment (garrypolley) [inline git-cart.sh]: What's this do? Comment above desired.

File: git-cart.sh (hunk 1 above)
Comment (Nate1729) [reply git-cart.sh]: Tells the computer which shell to run this with. `/bin/zsh` could also work. Without this line my current shell runs the script which may not be desired. For example, on my personal machine I like to use the `fish` shell which doesn't recognize this syntax.

Also happy to move this to a `/git-commands` folder.

File: git-cart.sh (hunk 2 above)
Comment (Nate1729) [reply git-cart.sh]: So I barely understand _how_ this works, but it just prints `| Repo | Current Branch` with some decent spacing so when this script prints the branches/repos underneath it everything lines up. The format specifier is the same when printing the actual repo lines.

**Comment by garrypolley** [comment]
I prefer a `gh` alias: 

```sh
gh alias set list-branches --shell 'printf "  %-20.20s | %-30.30s  \n\n" "Repo" "Current Branch" && gh repo list $GIT_ORG --json name --jq ".[].name" | xargs -L1 -I {} sh -c "cd $ORG_GIT_ROOT/{} 2> /dev/null && printf \"| %-20.20s | %-30.30s |\n\" \"{}\" \"$(git branch --show-current)\""'
```

This gets the branches -- but does so assuming a github based workflow.  I want to keep as much in `gh alias` as possible.

**Comment by garrypolley** [comment]
>  I want to keep as much in gh alias as possible

This is so I know how to check all the git benefits that have been added with: 

`gh alias list`

**Comment by garrypolley** [comment]
```sh
gh alias set list-branches --shell 'printf "  %-35.35s | %-50.50s  \n\n" "Repo" "Current Branch" && gh repo list $GIT_ORG --json name --jq ".[].name" | xargs -L1 -I {} sh -c "cd $ORG_GIT_ROOT/{} 2> /dev/null && echo {} && git branch --show-current" | xargs -n 2 | xargs -L1 -I {} sh -c "printf \"| %-35.35s | %-50.50s |\n\" {}"'
```

The prior version didn't properly call the `git branch --show-current` from the subshell. It was calling from the parent shell and always showing the current repo.  🤦 

Fixed the command: 

```sh
  Repo                                | Current Branch                                      

| terraform-applications              | master                                             |
| lumos                               | feature/sc-36025/create-new-all-cases-tab          |
| pensieve                            | master                                             |
| umbridge                            | master                                             |
| patronus                            | master                                             |
| dobby                               | master                                             |
| documentation                       | master                                             |
| wingardium                          | master                                             |
| user-management-testing             | master                                             |
| dumbledore                          | master                                             |
| sorcerer                            | master                                             |
| custom-metrics-service              | main                                               |
| terraform-modules                   | master                                             |
| ollivander                          | feature/local-docker-improvements                  |
| terraform-sso-internal-accounts     | master                                             |
| devops-tools                        | master                                             |
```

**Comment by garrypolley** [comment]
I'll close for now, selfishly preferring the alias mentioned in the comments. 😈

**Pull request closed without merging**